src/test/java/org/eolang/speco/SpecoEoTest.java

```diff
@@ -139,6 +140,7 @@ private static String[] dataize(final String target) throws IOException, Interru
             executor = "bash";
             flag = "-c";
         }
+        Logger.debug(this, String.format("Started compilation for %s", target));
```

Comment:
@advasileva In that case in the log you will see:
```
DEBUG org.eolang.speco.SpecoEoTest MDC= - Started compilation for /tmp/junit3607004110884547520/output
```
It doesn't tell something about the test case and about the code which will be transformed. Maybe we can add more context here? What do you think?

Reply:
@volodya-lombrozo I added the beginning of the test to the logs to make it clearer which test is currently being executed. In this test, I have only the text of the input program (even without the name of the test), so I didn't come up with anything better. I can also add a description of the test to `.yaml` test and outputs it as a context

Reply:
@volodya-lombrozo Done, created an issue #105.